### PR TITLE
MGDOBR-1384: add POST create sink connector feature

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-1384](https://issues.redhat.com/browse/MGDOBR-1384)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-XXX] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/SinkConnectorService.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    @Override
+    protected long getOrganisationConnectorsQuota(String organisationId) {
+        return quotaConfigurationProvider.getOrganisationQuotas(organisationId).getSinkConnectorsQuota();
+    }
+
+    @Override
+    protected List<Condition> createAcceptedConditions() {
+        List<Condition> conditions = new ArrayList<>();
+        conditions.add(new Condition(DefaultConditions.CP_KAFKA_TOPIC_CONNECTOR_READY_NAME, ConditionStatus.UNKNOWN, null, null, null, ComponentType.MANAGER, ZonedDateTime.now(ZoneOffset.UTC)));
```
Comment (manstis) [inline manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/SinkConnectorService.java:54]: I know it's not your doing; but IMO using a constructor with `null`'s suggests we should have a different constructor.

**Review by manstis — APPROVED** [review]
LGTM :+1: Nice refactoring of the tests.

File: manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/SinkConnectorService.java (hunk 1 above)
Comment (r00ta) [reply manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/SinkConnectorService.java:54]: Yep, I agree. I'll create a ticket for that :+1:

Comment (r00ta) [reply manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/SinkConnectorService.java:54]: https://issues.redhat.com/browse/MGDOBR-1406

**Pull request merged**